types/layout.d.ts

```diff
@@ -47,7 +47,7 @@ export interface LayoutItem {
 	/**
 	 * Draws the element
 	 */
-	draw(ChartArea): void;
+	draw(ChartArea: ChartArea): void;
```

Comment:
can we do this as `chartArea: ChartArea` since the params usually start with a lower case letter